clang/lib/CIR/CodeGen/CIRGenFunctionInfo.h

```diff
@@ -72,16 +68,15 @@ class RequiredArgs {
 
 class CIRGenFunctionInfo final
     : public llvm::FoldingSetNode,
-      private llvm::TrailingObjects<CIRGenFunctionInfo,
-                                    CIRGenFunctionInfoArgInfo> {
-  using ArgInfo = CIRGenFunctionInfoArgInfo;
-
+      private llvm::TrailingObjects<CIRGenFunctionInfo, CanQualType> {
```

Comment:
It would be nice if we have some comment here to indicate that `CanQualType` refers to the argument type.